apps/frontend/src/app/tournament-admin/tournament-form/tournament-form.component.html

```diff
@@ -0,0 +1,35 @@
+<form [formGroup]="myForm" (submit)="onSubmit()">
+  <div class="editTournamentData textAlign form-group">
+    <input matInput formControlName="name" id="editTournamentName" class="nameOfTournament" />
+    <div *ngIf="this.myForm.invalid">
+      <div class="error" *ngIf="this.myForm.get('name')?.hasError('required')">Name is required</div>
+      <div class="error" *ngIf="this.myForm.get('name')?.hasError('minlength')">Name must be at least 3 characters long.</div>
+      <div class="error" *ngIf="this.myForm.get('name')?.hasError('maxlength')">Name cannot have more than 40 characters.</div>
+    </div>
+    <hr style="width: 30%" />
+  </div>
+  <div id="tournamentData">
+    <label style="font-size: 1, 5em" for="description">Description:</label>
+    <textarea formControlName="description" style="width: 100%"></textarea>
```

Comment:
Also since you are using the angular material component library, please use their components for inputs and textareas.

You can find the documentation for them here: https://material.angular.io/components/form-field/overview